CHANGELOG.md

```diff
@@ -1,5 +1,83 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.0.5
+
+<!-- generated comparing v5.0.4..master -->
+
+_Oct 25, 2021_
+
+A big thanks to the 19 contributors who made this release possible. Here are some highlights ✨:
+
+Implement `Masonry` using Flexbox by @hbjORbj.
+Add three components to `@mui/core` by @rebeccahongsf and @hbjORbj.
```

Comment:
What problem would we solve? I had a couple at a couple of OSS projects to see what seems to work and what doesn't. The approach of React/Next.js is interesting, you get the changelog on GitHub and occasionally a blog post when it's a large release.

IMHO, we could first start writing more content on the changelog. It could be done by the PM or the dev rel on each team.